contracts/adapters/voting/OffchainVoting.sol

```diff
@@ -375,7 +375,7 @@ contract OffchainVotingContract is
         if (
             vote.gracePeriodStartingTime == 0 ||
             // check whether the new result changes the outcome
-            vote.nbNo > vote.nbYes != result.nbNo > result.nbYes
+            vote.nbNo > vote.nbYes != result.nbNo >= result.nbYes
```

Comment:
if (
            vote.gracePeriodStartingTime == 0 ||
            // check whether the new result changes the outcome
            vote.nbNo > vote.nbYes != result.nbNo > result.nbYes //it has changed from yes to no or no to yes 
            || ((vote.nbNo == vote.nbYes) != (result.nbNo == result.nbYes)) // was tie before and is not anymore or the opposite
        ) {
            vote.gracePeriodStartingTime = uint64(block.timestamp);
        }